### DISCUSSION TITLE
Pin apps to taskbar for a screen

### DISCUSSION THREAD

**Discussion by hillsurfer** [valinet/ExplorerPatcher#4070] [Ideas]
It would be great to be able to pin applications to the taskbars on screens other than the main screen. EP allows me to place those taskbars wherever I like, and it would be great to be able to pin things to those taskbars and have them open on that screen. Just a thought/wish. There may be other add-ons that would do this, but I'd rather stick to one. Thanks.

**Comment by pyrates999** [comment]
That is beyond what EP is for I think.
